packages/core/src/components/tabs2/tabs2.tsx

```diff
@@ -192,9 +208,9 @@ export class Tabs2 extends AbstractComponent<ITabs2Props, ITabs2State> {
         return undefined;
     }
 
-    /** Filters this.props.children to only `<Tab>`s */
-    private getTabChildren() {
-        return React.Children.toArray(this.props.children).filter(isTab) as TabElement[];
+    /** Filters children to only `<Tab>`s */
+    private getTabChildren(children = this.props.children) {
```

Comment:
This will fix your typings issues (we use this pattern elsewhere in the codebase):
```tsx
private getTabChildren(props: ITabs2Props & { children?: React.ReactNode } = this.props) {
    return React.Children.toArray(props.children).filter(isTab) as TabElement[];
}
```